<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zaxui</class>
 <widget class="QMainWindow" name="zaxui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1543</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-320</x>
      <y>-130</y>
      <width>1871</width>
      <height>971</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>material/Black_Template.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>690</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>material/B.G_Template_1.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>-150</y>
      <width>1161</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>material/Aqua.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(0, 0, 0);
background-image:url(:/newPrefix/s1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>250</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(0, 0, 0);
background-image:url(:/newPrefix/q1.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="terminalOutputBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>1551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
background-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="plainText">
     <string>bvcxcbx
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>material/sameer-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>terminalOutputBox</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
